ote_sdk/ote_sdk/usecases/exportable_code/demo/requirements.txt

```diff
@@ -1,3 +1,3 @@
 openvino==2022.1.0
 openmodelzoo-modelapi @ git+https://github.com/openvinotoolkit/open_model_zoo/@releases/2022/SCv1.1#egg=openmodelzoo-modelapi&subdirectory=demos/common/python
-ote-sdk @ git+https://github.com/openvinotoolkit/training_extensions/@561581d1f2118764237520a1520f46a1e9c87298#egg=ote-sdk&subdirectory=ote_sdk
+ote-sdk @ git+https://github.com/openvinotoolkit/training_extensions/@c29f1b385d5eabcf07bc9a6e9c340fd0dbf67c8d#egg=ote-sdk&subdirectory=ote_sdk
```

Comment:
As @goodsong81 suggestion, How about using branch name (i.e. `develop`) instead of commit number ?
If so, we could avoid the further mistakes

Reply:
- develop branch -> @develop (ensuring latest sync)
- release branches -> specific commit or tag (ensuring version match)
What do you think?

Reply:
At the other hand we could think about automatic commit switching for testing PRs, but this require some efforts at the CI side.

Reply:
@eugene123tw I agree, as you said if we want to fix the bug located in `OTE SDK`, using the branch name is hard to maintain. At this moment, let's use the current way and consider a better way to reduce the mistake that could be happened during the PR review.

Reply:
Thank you for the discussion and your thoughts. I replaced commit number with branch name